plugins/dialog/src/desktop.rs

```diff
@@ -226,5 +208,13 @@ pub fn show_message_dialog<R: Runtime, F: FnOnce(bool) + Send + 'static>(
     dialog: MessageDialogBuilder<R>,
     f: F,
 ) {
+    use rfd::MessageDialogResult as Result;
+    let f = move |res| {
+        f(match res {
+            Result::Ok | Result::Yes => true,
+            Result::Custom(s) => s == OK,
```

Comment:
This will still fail if the they used a custom ok button label, instead, you should match against `dialog.ok_button_label.unwrap_or_else(|| OK.into)`